### PR TITLE
Updating vSphere/OpenStack Cloud Providers - rancher_kubernetes_engine_config

### PR DESCRIPTION
**Pull request by sunilarjun**
Updating the vSphere and OpenStack cloud provider pages with more information on the `rancher_kubernetes_engine_config` directive and where to update the cluster config YAML file.

Fixes https://github.com/rancher/rancher-docs/issues/140

### CODE REVIEW

File: docs/config-options/cloud-providers/openstack/openstack.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: OpenStack Cloud Provider
 ---
 
-To enable the OpenStack cloud provider, besides setting the name as `openstack`, there are specific configuration options that must be set. The OpenStack configuration options are grouped into different sections.
+To enable the OpenStack cloud provider, besides setting the name as `openstack`, there are specific configuration options that must be set as defined below. The OpenStack configuration options are grouped into different sections and they must be nested under the `rancher_kubernetes_engine_config` directive in the cluster config YAML file. For more information on the configuration file structure please see the [Rancher RKE Cluster Configuration Reference](https://ranchermanager.docs.rancher.com/reference-guides/cluster-configuration/rancher-server-configuration/rke1-cluster-configuration#config-file-structure-in-rancher).
```
Comment (LucasSaintarbor) [inline docs/config-options/cloud-providers/openstack/openstack.md:5]:
```suggestion
To enable the OpenStack cloud provider, besides setting the name as `openstack`, specific configuration options must be set as defined below. The OpenStack configuration options are grouped into different sections and they must be nested under the `rancher_kubernetes_engine_config` directive in the cluster config YAML file. For more information on the configuration file structure, please see the [Rancher RKE Cluster Configuration Reference](https://ranchermanager.docs.rancher.com/reference-guides/cluster-configuration/rancher-server-configuration/rke1-cluster-configuration#config-file-structure-in-rancher).
```

File: docs/config-options/cloud-providers/vsphere/vsphere.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ This section describes how to enable the vSphere cloud provider. You will need t
 
 The [vSphere Cloud Provider](https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/) interacts with VMware infrastructure (vCenter or standalone ESXi server) to provision and manage storage for persistent volumes in a Kubernetes cluster.
 
-When provisioning Kubernetes using RKE CLI or using [RKE clusters](https://ranchermanager.docs.rancher.com/pages-for-subheaders/launch-kubernetes-with-rancher) in Rancher, the vSphere Cloud Provider can be enabled by configuring the `cloud_provider` directive in the cluster YAML file.
+When provisioning Kubernetes using RKE CLI or using [RKE clusters](https://ranchermanager.docs.rancher.com/pages-for-subheaders/launch-kubernetes-with-rancher) in Rancher, the vSphere Cloud Provider can be enabled by configuring the `cloud_provider` directive nested under the `rancher_kubernetes_engine_config` directive in the cluster config YAML file. For more information on the configuration file structure please see the [Rancher RKE Cluster Configuration Reference](https://ranchermanager.docs.rancher.com/reference-guides/cluster-configuration/rancher-server-configuration/rke1-cluster-configuration#config-file-structure-in-rancher).
```
Comment (LucasSaintarbor) [inline docs/config-options/cloud-providers/vsphere/vsphere.md:9]:
```suggestion
When provisioning Kubernetes using RKE CLI or using [RKE clusters](https://ranchermanager.docs.rancher.com/pages-for-subheaders/launch-kubernetes-with-rancher) in Rancher, the vSphere Cloud Provider can be enabled by configuring the `cloud_provider` directive nested under the `rancher_kubernetes_engine_config` directive in the cluster config YAML file. For more information on the configuration file structure, please see the [Rancher RKE Cluster Configuration Reference](https://ranchermanager.docs.rancher.com/reference-guides/cluster-configuration/rancher-server-configuration/rke1-cluster-configuration#config-file-structure-in-rancher).
```

**Review by LucasSaintarbor — APPROVED** [review]
Small NITs. LGTM 👍

File: docs/config-options/cloud-providers/openstack/openstack.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 title: OpenStack Cloud Provider
 ---
 
-To enable the OpenStack cloud provider, besides setting the name as `openstack`, there are specific configuration options that must be set. The OpenStack configuration options are grouped into different sections.
+To enable the OpenStack cloud provider, besides setting the name as `openstack`, specific configuration options must be set as defined below. The OpenStack configuration options are grouped into different sections and they must be nested under the `rancher_kubernetes_engine_config` directive in the cluster config YAML file. For more information on the configuration file structure, please see the [Rancher RKE Cluster Configuration Reference](https://ranchermanager.docs.rancher.com/reference-guides/cluster-configuration/rancher-server-configuration/rke1-cluster-configuration#config-file-structure-in-rancher).
```
Comment (btat) [inline docs/config-options/cloud-providers/openstack/openstack.md:5]: The directive is only for the Rancher case, not the standalone RKE case.

File: docs/config-options/cloud-providers/vsphere/vsphere.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ This section describes how to enable the vSphere cloud provider. You will need t
 
 The [vSphere Cloud Provider](https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/) interacts with VMware infrastructure (vCenter or standalone ESXi server) to provision and manage storage for persistent volumes in a Kubernetes cluster.
 
-When provisioning Kubernetes using RKE CLI or using [RKE clusters](https://ranchermanager.docs.rancher.com/pages-for-subheaders/launch-kubernetes-with-rancher) in Rancher, the vSphere Cloud Provider can be enabled by configuring the `cloud_provider` directive in the cluster YAML file.
+When provisioning Kubernetes using RKE CLI or using [RKE clusters](https://ranchermanager.docs.rancher.com/pages-for-subheaders/launch-kubernetes-with-rancher) in Rancher, the vSphere Cloud Provider can be enabled by configuring the `cloud_provider` directive nested under the `rancher_kubernetes_engine_config` directive in the cluster config YAML file. For more information on the configuration file structure, please see the [Rancher RKE Cluster Configuration Reference](https://ranchermanager.docs.rancher.com/reference-guides/cluster-configuration/rancher-server-configuration/rke1-cluster-configuration#config-file-structure-in-rancher).
```
Comment (btat) [inline docs/config-options/cloud-providers/vsphere/vsphere.md:9]: The directive is only for the Rancher case, not the standalone RKE case.

The standalone RKE case has a dedicated section (L24), maybe we can follow that pattern here too.

**Review by btat — CHANGES_REQUESTED** [review]
Left a couple comments on the current proposed changes.

Unrelated to this PR, but since the file is being updated here...the last sentence on L26 of the vSphere file should probably be adjusted to be a bit more professional.

Also, I'm not sure how appropriate/necessary it is to include all of these Rancher specific bits in the standalone RKE docs (similar to the request leading to this PR), but let's discuss outside this PR since looking at the existing content suggests the waters are already muddy around that topic.

File: docs/config-options/cloud-providers/vsphere/vsphere.md (hunk 4 above)
Comment (sunilarjun) [reply docs/config-options/cloud-providers/vsphere/vsphere.md]: Hey Billy thanks for the review! I revised both pages and revised the vSphere layout a bit to hopefully better the readability for users. Let me know how the changes look, thanks!

**Review by btat — APPROVED** [review]

**Pull request merged**